docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -28,6 +28,9 @@ Setup Dataflow
 
 .. include:: ../../includes/dataflow_on_gce_setup.rst
 
+Your project must be whitelisted to use gRPC in order to run this pipeline, as it makes use of 
+gRPC streaming to retrieve data.
```

Comment:
Is there a link we can add here so that if people are interested, they can request to be whitelisted?